Dockerfile.deps

```diff
@@ -32,11 +32,11 @@ RUN curl -sL -o bitcoin.tar.gz https://bitcoincore.org/bin/bitcoin-core-0.20.1/b
  && ln -s /srv/explorer/bitcoin-0.20.1 /srv/explorer/bitcoin \
  && rm bitcoin.tar.gz
 
-ENV SHA256SUM_ELEMENTS=d6c94c323b9c586f46b10ab79aa643342b950fd0f01916a47d8051036e6a2288
-RUN curl -sL -o elements.tar.gz https://github.com/ElementsProject/elements/releases/download/elements-0.18.1.11/elements-0.18.1.11-x86_64-linux-gnu.tar.gz \
+ENV SHA256SUM_ELEMENTS=b4038a34fcb3f1ed55acc0891cea8cd988ac8f164be2bb93d6dee24779744fae
+RUN curl -sL -o elements.tar.gz https://github.com/ElementsProject/elements/releases/download/elements-0.18.1.12/elements-0.18.1.12-x86_64-linux-gnu.tar.gz \
```

Comment:
Nit - maybe we should add an ELEMENTS_VER=0.18.1.12 to simplify updates.

Same for BITCOIN_VER above?